exercises/concept/bettys-bike-shop/.docs/introduction.md

```diff
@@ -0,0 +1,105 @@
+# Haskell Syntax Basics 2
```

Comment:
This introduction is quite big. As per [the docs](https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-introduction-md), the introduction document should only contain information that is needed to understand the fundamentals of the concept and solve the exercise.

Reply:
To me, it also looks like this exercise might be doing a bit too much. It is introducing: 

- Modules
- Imports
- Exports
- Ints and Float
- Strings

I think this is too much for a single exercise. There are several options to fix this issue:

1. Make this exercise solely about modules and add prerequisites for the `integers`, `floating-point-numbers` and `strings` concepts.
2. Make this exercise about modules and strings and add prerequisites for the `integers` and `floating-point-numbers` concepts.
3. Don't use floating-point-numbers in the exercise and just use integers. Normally, you'd still want to add a `integers` prerequisite. However, as the `basics` concept gives students a minimal understanding of integers, you could get away with not having that here (this is an exception though).
4. Don't use floats or integers at all and just have strings.

Some of the above options could even be combined.

Reply:
Ok, I will make the introduction for the exercises to only what is needed for the exercise, and break-up the basics-2 concept into integers, floating-point-numbers and strings. I will also break-up the exercise to use only the concepts introduced to that point, unless I find already made exercise for the Int and Float concepts.